### PR TITLE
Plantation fields schematic information

### PR DESCRIPTION
**Pull request by Thodor12**
Adds the required schematic information for plantation fields

### CODE REVIEW

File: pages/src/source/tutorials/schematics.md [hunk 1]
```diff
@@ -341,8 +347,115 @@ You should always apply a [groundlevel tag](../../source/items/tagtool) when mak
...
+
+The amount of planting positions on these fields are usually unlimited, this is because Mojang directly affects how much growth a bonemeal action can already do, although I wouldn't make the fields too big for no reason, if you make patches of at most 7x7 you should be fine.
+
+This one requires a bit more detail as every plant works slightly different.
+
+| Plant          | Work tag location                                                                                                                                                                       |
+|----------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Seagrass       | For seagrass, the block below the water line itself should be tagged, the planter is supposed to bonemeal the dirt block itself.                                                        |
+| Sea pickles    | For sea pickles, the block below the water line itself should be tagged, however the planter will initially place the pickles, and then bonemeal the pickles directly to let them grow. |
```
Comment (uecasm) [inline pages/src/source/tutorials/schematics.md:447]: These could benefit from an image... is "the block below the water line" the water block itself, or the ground block underwater?

**Comment by uecasm** [comment]
Afterthought: I wonder if some of the specific tag details ought to be on the [tag tool page](https://github.com/ldtteam/MinecoloniesWiki/blob/main/pages/src/source/items/tagtool.md) instead, since that's where all the other tags are discussed.  Certainly the plantation tags mentioned there are now outdated.

**Comment by Thodor12** [comment]
The tags on the plantation are still applicable, for now at least, because the old plantation still serves as a "field" for sugar cane, cactus and bamboo

**Comment by Talyda** [comment]
Having just had a look at the page, splitting it might be a good idea! It really is getting very long!
Maybe, as you already have an index on that page, you could use that to split it a bit? But with it making a bit more sense, cos having "plantation fields" as it's own category there in the index, while mines and quarries are squished in "additional information" doesn't really make sense for headers/page splits

**Pull request merged**